var/spack/repos/builtin/packages/gnina/package.py

```diff
@@ -68,13 +68,31 @@ class Gnina(CMakePackage, CudaPackage):
     depends_on("py-numpy", type="build")
     depends_on("py-pytest", type="build")
 
-    depends_on("cuda@11")
+    depends_on("cuda@11", when="@:1.1")
+    depends_on("cuda@12", when="@1.3:")
     depends_on("cudnn", when="+cudnn")
 
+    depends_on("cmake@3.27:", when="@1.3:")  # CMake policy CMP0146 introduced in 3.27
+    depends_on("jsoncpp", when="@1.3")
+    depends_on("py-torch", when="@1.3")
+
+    depends_on("glog@:0.6", when="@:1.1")
+    depends_on("hdf5+cxx+hl~mpi", when="@:1.1")
+    depends_on("openblas~fortran", when="@:1.1")
+
+    patch("PR280-v1.3.patch", when="@1.3")
+    patch("PR282-v1.3.patch", when="@1.3")
+
     def cmake_args(self):
-        args = ["-DBLAS=Open"]  # Use OpenBLAS instead of Atlas' BLAS
+        args = []
+
+        if self.spec.satisfies("@:1.1"):
+            argg.append("-DBLAS=Open")  # Use OpenBLAS instead of Atlas' BLAS
 
         if self.spec.satisfies("+gninavis"):
             args.append(f"-DRDKIT_INCLUDE_DIR={self.spec['rdkit'].prefix.include.rdkit}")
 
+        if self.spec.satisfies("@1.3:"):
+            args.append("-DGNINA_FORCE_EXTERNAL_LIBS=ON")
```

Comment:
Optional:
```suggestion
            args.append(self.define("GNINA_FORCE_EXTERNAL_LIBS", True)
```
No big practical difference, but reads a bit better and defines the variable as boolean in CMake.